gems/pending/openstack/openstack_handle/handle.rb

```diff
@@ -32,7 +32,14 @@ def self.try_connection(security_protocol)
         # For backwards compatibility take blank security_protocol as SSL
         yield "https", {:ssl_verify_peer => false}
       elsif security_protocol == 'ssl-with-validation'
-        yield "https", {:ssl_verify_peer => true}
+        excon_options = {:ssl_verify_peer => true}
+        ssl_ca_file = VMDB::Config.new("vmdb").config.fetch_path(:ssl, :ssl_ca_file)
+        ssl_ca_path = VMDB::Config.new("vmdb").config.fetch_path(:ssl, :ssl_ca_path)
```

Comment:
These values should be passed in as parameters.  Classes in `gems/pending` shouldn't depend on `VMDB::Config` directly.  Maybe include them as constructor values?